Chinese, Simplified Localization (zh-Hans)/devicePropertiesController.strings

```diff
@@ -1,41 +1,34 @@
-/* Class = "NSView"; ibShadowedToolTip = "2. Block\nType: plist dict\nDescription: Removes device properties from a map (plist dict) of deivce paths to an array (plist array) of variable names in plist string format."; ObjectID = "1eh-jH-TX2"; */
-"1eh-jH-TX2.ibShadowedToolTip" = "2. Block\nType: plist dict\nDescription: Removes device properties from a map (plist dict) of deivce paths to an array (plist array) of variable names in plist string format.";
-
 /* Class = "NSTableColumn"; headerCell.title = "Value"; ObjectID = "6on-lJ-yiP"; */
 "6on-lJ-yiP.headerCell.title" = "值";
 
+"dpG-4m-Tya.headerCell.title" = "类型";
+
 /* Class = "NSTableColumn"; headerCell.title = "Devices"; ObjectID = "9kg-WA-Eir"; */
 "9kg-WA-Eir.headerCell.title" = "设备";
 
 /* Class = "NSTextFieldCell"; title = "Text Cell"; ObjectID = "CrE-0l-DLY"; */
 "CrE-0l-DLY.title" = "Text Cell";
 
-/* Class = "NSTableColumn"; headerCell.title = "Type"; ObjectID = "dpG-4m-Tya"; */
-"dpG-4m-Tya.headerCell.title" = "Type";
-
-/* Class = "NSTextFieldCell"; title = "Text Cell"; ObjectID = "eia-C9-4Iw"; */
-"eia-C9-4Iw.title" = "Text Cell";
-
 /* Class = "NSTableColumn"; headerCell.title = "Key*"; ObjectID = "F6A-yy-QfQ"; */
 "F6A-yy-QfQ.headerCell.title" = "键值*";
 
-/* Class = "NSMenuItem"; title = "NUMBER"; ObjectID = "lI7-XR-ash"; */
-"lI7-XR-ash.title" = "NUMBER";
-
```

Comment:
DATA, STRING, NUMBER are menu items. Required!

Reply:
OK, I resubmit the changes!